### ISSUE TITLE
PSX games - On Retroarch it works but not on ES

### ISSUE THREAD

**Issue by Tunaa** [Aloshi/EmulationStation#662]
I choose the core and the bios directory for a psx game and it works on retroarch. But if i want to play the game in ES it goes back to the menu.

**Comment by DavidinCT** [comment]
Check your config files... copy the exact command and see if it works in a RUN command box...

These problems come from the es_systems file...

**Comment by Tunaa** [comment]
yeah my config was messed up. the bios path was wrong. but could you please show me an example how i run a command in the command box?
